_posts/2019/2019-05-04-react-hooks-what-all-the-fuss-is-about.md

```diff
@@ -0,0 +1,57 @@
+---
+layout: post
+title: React Hooks - What all the fuss is about
+date: 2019-05-04
+header_image: public/react-hooks.jpg
+header_position: center
+category: coding
+tags: ["javascript", "reactjs"]
+authors: ["Jonathan"]
+about_authors: ["jwieben"]
+---
+
+## React Hooks - What all the fuss is about
+
+Reacts new Hooks have been all the rage these last few months. You might have heard about them without really understanding what they are and why so many people are so excited about them. Maybe you have asked yourself why a simple API proposal would get [thousand of comments and reactions on its RFC](https://github.com/reactjs/rfcs/pull/68){:target="_blank"}. This post intends to help you understand some of the hype.
```

Comment:
I tend to write "hooks" (lowercase) here. In the headline it's fine though.

thousands (add an s) of comments...